### DISCUSSION TITLE
automatic fixtures for the relevant accounts

### DISCUSSION THREAD

**Discussion by benkoshy** [janko/rodauth-rails#120] [Ideas]
When the migrations are run, accounts are created, would it not be handy if fixtures were also created (as with rails)? Some might be using rspec, so the benefits are moot. 

I can try for a PR on this later on though, if it's worth it.

**Comment by janko** [comment]
I can imagine the `rodauth:install` generator creating a fixture for the `Account` model, but I don't think that creating fixtures for other tables would be useful enough, as I think people will rarely create these records in tests.
